docker-compose.yml

```diff
@@ -56,32 +58,39 @@ services:
       - inferno_program
       - fhir_validator_app
       - community_validator_service
+    mem_limit: 100m
+    cpuset: '3,4'
```

Comment:
If `cpuset` is zero-indexed, like https://github.com/onc-healthit/inferno.healthit.gov/pull/76/files#diff-e45e45baeda1c1e73482975a664062aa56f20c03dd9d64a827aba57775bed0d3R19 implies, should this be `cpuset: '2,3'`?